quantum/split_common/split_util.c

```diff
@@ -135,6 +132,9 @@ __attribute__((weak)) bool is_keyboard_master(void) {
 void split_pre_init(void) {
     isLeftHand = is_keyboard_left();
 
+    // Start in slave configuration and wait for a master to emerge
+    transport_slave_init();
```

Comment:
Adding this here now calls init twice, and breaks #8235 again.

Reply:
You mean, it's called twice for the master half?

Reply:
I guess, for changing the target branch without strange side effects, I need to reopen this.

Reply:
no slave half <https://github.com/qmk/qmk_firmware/blob/72e522bfc6c98f0710b1c2fb615b600730aa96cb/quantum/split_common/split_util.c#L158-L162>

Reply:
Do you think that if I just remove the repeated initialization in `split_post_init`, it will bring back the race-condition bug? The slave will finish USB initialization when the master is determined and will not overwrite any settings that are sent over the transport.